main.py

```
@@ -10,349 +9,19 @@
-        logger.info(
-            f"Valid short-term expirations for {ticker_symbol}: {valid_expirations}"
-        )
-        return valid_expirations
-    except Exception as e:
-        logger.warning(
-            f"Could not fetch options data for {ticker_symbol}: {e}")
-        return []
 
+from logging_setup import setup_logging
+from openai_client import setup_openai_client
+from data_fetching import fetch_sp500_tickers, get_financial_statements, get_gross_margins_quarters
```

Ruff (0.8.2)

`data_fetching.get_financial_statements` imported but unused

Remove unused import: `data_fetching.get_financial_statements`

(F401)